packages/tasks/src/pipelines.ts

```diff
@@ -652,6 +652,11 @@ export const PIPELINE_DATA = {
 		modality: "cv",
 		color: "indigo",
 	},
+	"video-text-to-text": {
+		name: "Video-Text-to-Text",
+		modality: "multimodal",
+		color: "blue",
```

Comment:
```suggestion
		color: "blue",
		hideInDatasets: true,
```

As mentioned, I think we can merge if `hideInDatasets` is added. Do you see a problem with it @merveenoyan?